### ISSUE TITLE
check urls: failing urls to fix or skip

### ISSUE THREAD

**Issue by NeilHanlon** [rocky-linux/wiki.rockylinux.org#12]
```
🤔 Uh oh... The following urls did not pass:
https://freenode.net/policies
https://docs.rockylinux.org/release_notes/
https://example.com/cache
https://wiki.rockylinux.org/en/irc-mattermost-bridge
https://git.centos.org/rpms/nginx**
https://wiki.rockylinux.org/en/team/development/debranding/how-to
https://pagure.io/includedmodule.git
https://pagure.io/bar.git
https://linkedin.com/company/rockylinux
```

would be good to get these skipped or fixed. To skip, we can add to the configuration for the urlchecker in `.github/workflows/test.yml`. see https://github.com/urlstechie/urlchecker-action

**Comment by NezSez** [comment]
It validates links, but doesn't report the source where the link was used? That seems like a major shortcoming; the script already has the info of where it got the URL from and should have included that in the output to make fixing links quicker. Now one has to have access to any files it scraped, and time to grep/regex/parse thru them just to be able to check them. Silly.

Anything in "example.com" (et. al. see RFC link below) should be skipped as those domain were created explicitly to be used in documentation, home or local networks and cannot actually be used/bought/registered for commercial or non-profit use. For the record, it's the same for home.arpa, which officially ppl should use in personal local networks (where many use "local.net", "localnet", etc). These and a few other domains are listed in the [IETF DNS RFC 6761](https://www.rfc-editor.org/rfc/rfc6761.html).

https://linkedin.com/company/rockylinux just needs the www added to work ( https://www.linkedin.com/company/rockylinux )

https://git.centos.org/rpms/nginx** just needs the ** removed to work, assuming that was the intent

**Comment by NeilHanlon** [comment]
Yeah, it's a bit frustrating that it doesn't output the filename at the same time. That might be an option but I've not found it. There is an output with filename + location here: https://github.com/rocky-linux/wiki.rockylinux.org/runs/6185552435?check_suite_focus=true#step:4:45

but obviously not ideal :)
